README.md

```diff
@@ -26,6 +26,8 @@ your disk.
 |`-j, --jobs`|Number of threads to use|
 |`-o, --output`|Output directory where the downloaded files are written|
 |`-t, --tries`|Number of times to retry when the downloading of a page fails|
+|`-u, --user-agent`|User agent to be used for sending requests|
+|`-w, --wildcards`|Wildcards to filter pages to only matched ones|
```

Comment:
This should be `include` and `exclude` now

Reply:
✔️